tools/refguide_check.py

```diff
@@ -55,13 +57,23 @@ def get_all_dict(module):
         except ValueError:
             pass
 
+    # FIXME: shouldn't modules be in the refguide, actually? if they're in __all__?
     # somehow some modules survive the first iteration (?)
     for _ in range(2):
         for name in all:
             if inspect.ismodule(getattr(module, name)):
                 all.remove(name)
 
-    return all
+    deprecated = []
+    for name in all:
+        f = getattr(module, name)
+        if name in ['info', 'who']: # These print to stdout when called
```

Comment:
Here's a way that numpy redirects `who` to `/dev/null` in a test:
https://github.com/numpy/numpy/blob/maintenance/1.9.x/numpy/lib/tests/test_regression.py#L171